styles/prosilver/template/common/contribution_details.html

```diff
@@ -63,6 +63,12 @@
 							<dd>{downloads.INSTALL_LEVEL}</dd>
 						</dl>
 						<!-- ENDIF -->
+						<!-- IF downloads.INSTALL_TO -->
```

Comment:
Why are we using old PHPBB syntax instead of Twig? I think we should do Twig (in fact we should change it for the whole extension but that will be another PR).

Reply:
I thought about that too and also now with the old language function, but I also think it's best to tackle both of those things separately in their own PR's.

Reply:
Yes separate PR (I'm working on that now). But lets please make all new code use Twig syntax 😄

Reply:
No problem, I've changed it to Twig